.github/CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 *                     @mjcarroll
+tutorials/*           @maryaB-osr
```

Comment:
I think this is a bad merge, the line should be removed as of https://github.com/ignitionrobotics/ign-common/pull/88/

Reply:
Fixed in the merge commit and force-pushed.